cmd/client-nfd/main.go

```diff
@@ -0,0 +1,27 @@
+/*
+Copyright 2024 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"sigs.k8s.io/node-feature-discovery/cmd/client-nfd/subcmd"
```

Comment:
Let's try to handle as we do for `nfd-master` and `nfd-worker` to maintain the project structure. (see how we hande flags and sub commands for those 2

Reply:
Right, I was too fast. Will do the same as you do for `nfd-master` and `nfd-worker`.

Reply:
@ArangoGutierrez one question: aside from the `platform` option, I followed the style used in the [kubectl cmd](https://github.com/kubernetes-sigs/node-feature-discovery/tree/master/cmd/kubectl-nfd). Should the kubectl cmd also be refactored to align with the `nfd-master` and `nfd-worker` styles? I could open a separate PR for that.

Reply:
Let's create an issue for that and set as a TODO, but let's focus on getting this one in